Fixing latency issue found in remote control fetched in vla_eval

ameesh-shah opened the pull request:
This PR makes the following change:

* Identifies and fixes an issue in vla_eval.py where a latency issue in fetching images from the robot-side server resulted in inconsistent control output from the model-side client.
* The fix starts a concurrent thread that persistently refreshes the image input and corrects the control output as a result.
* Wrappers are added that are important for openvla performance (ConvertState2Proprio, ResizeObsImageWrapper)
* The configuration is reset upon manual termination.

youliangtan commented on policies/vla_eval.py:
```diff
@@ -95,50 +101,73 @@ def main(_):
     # format the prompt
     prompt = f"In: What action should the robot take to {FLAGS.text_cond}?\nOut:"
+    print(prompt)
+    def run_continuous_img_primary(manipulator_interface):
```
Can we move this into the `ActionClientInterface()` impl? so this bug fix can be applied to others that are using the `ActionClientInterface` api

youliangtan commented:
@ameesh-shah Thanks for identifying this fix with @zhouzypaul . I think there might be some deeper problem within the `ret, frame = self._caps[0].read()` within the `interfaces.widowx.py`, where the frame got cache, (aka not the latest img). So might need to do threading on the server side, so we always store the latest frame.

zhouzypaul commented:
Yea the `self._caps[0].read()` reads the next frame from the camera's buffer and each call to read() advances the buffer to the next frame. It is essential to call this to update the buffer. Or else there will be latency issue, especially when the action client is not hosted on the same machine as the action server, I think. 

I'll work with @ameesh-shah to fix the interfaces implementation for a general fix.

ameesh-shah commented:
Added the fix with @zhouzypaul to persistently fetch updated images across all of the manipulator interfaces in `interfaces`, and removed this redundant logic in `vla_eval.py`. The PR should be ready for review.

zhouzypaul commented:
Looks good to me! @youliangtan

youliangtan reviewed (approved):
Thanks ~ this looks good to me!
